templates/layout/google_tag_manager.html

```diff
@@ -6,14 +6,7 @@
       (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
         new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
         j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;
-
-        j.addEventListener('load', function() {
-          // notify listeners about loaded gtm
-          d.dispatchEvent(new CustomEvent('gtm_loaded', { bubbles: true }));
```

Comment:
gtm_loaded event is not required for STB too, i suppose.
If it's true, create plz pdd subtask to delete it from STB